docs/docs/policies/actions.md

```diff
@@ -22,26 +22,98 @@ rules:
   - event: dropped_executable
 ```
 
-### Webhook action
+if we start tracee it will log table to stdout:
+
+```
+tracee --policies policy.yaml
+```
+
+To customize it, we use the [`--output`](../../outputs/output-formats) flag, for example, instead of table I would like to log a json to stdout:
```

Comment:
```suggestion
To customize it, we use the [`--output`](../../outputs/output-formats) flag. For example, instead of table I would like to log a json to stdout:
```